### DISCUSSION TITLE
LMEEF

### DISCUSSION THREAD

**Discussion by TheActualJacob** [3kh0/ext-remover#637] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
LTMEAT worked by removing the filtering page for filtered websites but a connection established error shows up instead. However, things like YouTube videos are uncensored. All blocked sites won't load but unblocked sites do load.

**Comment by jessiejs** [comment]
d-do you happen to be using school wi-fi?

**Reply by TheActualJacob** [reply]
Ah okay. I tried my hotspot and it works. So I'm guessing there's no way to bypass the schools network proxy other than using a hotspot right?

**Reply by CheezeKake77** [reply, marked as answer]
You can use a dns/vpn on your chromebook if you happen to have one

**Reply by TheActualJacob** [reply]
I can't do it on school Wi-Fi though.
